.github/workflows/ci-rust.yml

```diff
@@ -161,7 +161,37 @@ jobs:
 
       - name: Test platform-device-loader on wasm
         if: inputs.run-wasm-tests
-        run: wasm-pack test --headless --firefox libparsec/crates/platform_device_loader
+        shell: bash -o pipefail -ex {0}
```

Comment:
Should be reverted to launching all tests (or there should be a comment explaining what's going on here ^^)